src/window.ui

```diff
@@ -70,5 +57,8 @@
         </child>
       </object>
     </child>
+    <child>
+      <object class="GtkEventControllerKey" id="eventController"/>
+    </child>
```

Comment:
I don't mind either way; Is there any benefit to creating this object in the ui file?

Reply:
Slightly less code to write on the JS side.